js/kanji.js

```diff
@@ -150,6 +151,52 @@ window.Kanji =  {
     }
   },
 
+  _initialSelectAllHide: function() {
+    $(".category-select-all").each(function() {
+      $(this).hide();
+    });
+    $(".category-deselect-all").each(function() {
+      $(this).hide();
+    });
+    $(".category-select-all-show").each(function() {
```

Comment:
It would be great to change these to single quotes like the rest of the code base.